test/topic.ts

```diff
@@ -164,7 +168,7 @@ describe('Topic', () => {
     });
 
     it('should localize the request function', done => {
-      PUBSUB.request = callback => {
+      PUBSUB.request = (callback: RequestCallback<google.protobuf.Empty>) => {
```

Comment:
I mean, that or just `Function` here is fine to be honest.  I'm ok with the types being a tad sloppy in the unit tests.